packages/medusa/src/api/routes/admin/orders/list-orders.js

```diff
@@ -15,6 +15,18 @@ export default async (req, res) => {
       selector.q = req.query.q
     }
 
+    if ("payment_status" in req.query) {
+      selector.payment_status = req.query.payment_status
```

Comment:
would recommend doing this instead:

`selector.payment_status = req.query.payment_status.split(",")`

That way we enable queries for multiple statusses.